src/modules/notification/infrastructure/store/modules/notification.ts

```diff
@@ -0,0 +1,44 @@
+import EventNotification from '../../../domain/entity/EventNotification';
+import { ActionTree, createStore, GetterTree, MutationTree } from 'vuex';
+import { Actions, ActionTypes, Getters, Mutations, MutationTypes, State, Store } from '../types';
+
+const state = ():State => ({
```

Comment:
```suggestion
const state = (): State => ({
```

Also, `State` is too ambiguous.